gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -139,22 +134,17 @@ class <%= service.client_name %>
     )
   end
 
-<%- if service.lro? -%>
+  <%- if service.subclients? -%>
+  <%- service.subclients.each do |subclient| -%>
   ##
-  # Get the associated client for long-running operations.
+  # Get the associated client for <%= subclient.service_description %>.
   #
-  # @return [<%= service.operations_name_full %>]
+  # @return [<%= subclient.client_class_docname %>]
   #
-  attr_reader :<%= service.lro_client_var %>
-
-<%- end -%>
-<%- if service.mixins? -%>
-  <%- service.mixins.each do |mixin| -%>
-  # @return [<%= mixin.client_class_name %>]
-  attr_reader :<%= mixin.client_var_name %>
+  attr_reader :<%= subclient.client_var_name %>
 
   <% end %>
-<% end %>
+  <% end %>
```

Comment:
We should change this and the previous line to `<%- end -%>` to eliminate extra whitespace.

Reply:
:+1: